docs/source/deployment/aws_ecs.rst

```diff
@@ -86,8 +91,26 @@ Use the IrisClassifier BentoService from the :doc:`Quick start guide<../quicksta
     }
 
 
+The BentoML saved bundle created can now be used to start a REST API Server hosting the
+BentoService and available for sending test request:
+
+.. code-block:: bash
+
+    # Start BentoML API server:
+    bentoml serve IrisClassifier:latest
+
+
+.. code-block:: bash
+
+    # Send test request:
+    curl -i \
+      --header "Content-Type: application/json" \
+      --request POST \
+      --data '[[5.1, 3.5, 1.4, 0.2]]' \
+      http://localhost:5000/predict
+
 ==================================================================
-Build and push docker image to AWS ECR(Elastic Container Registry)
+Build BentoML model server for AWS ECR(Elastic Container Registry)
```

Comment:
change title to: Dockerize BentoML model server for deployment

And explain to create ECS deployment, it needs to build docker image and push the image to ECR.

Reply:
can you also update the dockeriation steps below to the new one we've been using with `bentoml get ... | jq ...`